.env.template

```diff
@@ -132,8 +132,10 @@ GDS_BFF_CONSOLE_LOG=true
 GDS_BFF_ALLOW_ORIGINS="http://localhost:3000,http://localhost:3003"
 GDS_BFF_COOKIE_DOMAIN=localhost
 
-GDS_BFF_AUTH0_ISSUER=
+GDS_BFF_AUTH0_DOMAIN=
```

Comment:
The domain is required to compute the management API endpoints - so we could no longer call this the issuer. This might change some .env files for folks who have been running BFF in docker compose.